src/Web/Controllers/AccountController.cs

```diff
@@ -117,18 +118,19 @@ private async Task<IActionResult> SignInUser(LoginViewModel model, string return
                 {
                     UserType = AccountProtection.UserType.Consumer,
                     Username = model.Email,
-                    PasswordHash = hashedPassword
+                    UserId = model.Email
```

Comment:
Does setting this UserId have any implications?  e.g. will it create a User node in the graph or anything weird even if the SignIn is rejected?  What if the SignIn succeeds?  Does it create a user node in either case?

Reply:
UserId was changed to be a required property.  It is anonymized and sent downstream, but I'm not aware of any implications beyond that.

Regarding graph nodes, I'm not sure how this is working in the new AP world.  The calls aren't going through KG anymore, which would ordinarily be the service responsible for that.